File: Python/D18 - M/5/5.py
# Write a python program to find maximum and minimum of two numbers using module.

import min_num, max_num

x = int(input('Enter number 1 : '))
y = int(input('Enter number 1 : '))

if(x != y):
    min_num.min_number(x,y)
    max_num.max_number(x,y)
else:
    print('Both numbers are equal')

File: C/D2/5 - nestedIfElse/5.cpp
// Write a C program to find max number from 3 numbers using nested if else statement.

#include <stdio.h>

int main(){
    int a, b, c;

    printf("Enter values : ");
    scanf("%d %d %d", &a, &b, &c);

    if(a>b){
        if(a>c){
            printf("Max number is %d", a);
        }
        else{
            printf("Max number is %d", c);
        }
    }
    else{
        if(b>c){
            printf("Max number is %d", b);
        }
        else{
            printf("Max number is %d", c);
        }
    }
}